src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+
+Live migration of RBD images in Ceph allows you to move storage seamlessly between
+pools, RBD namespaces, and clusters in different formats with minimal downtime.
+Let's break it down into three simple steps, along with the necessary commands to execute them.
+
+#### Step 1: Prepare for Migration
+
+In this step before starting the migration, a new target image is created and
+linked to the source image.
+
+* If import-only mode is not enabled, the source image will be marked as read-only to prevent modifications.
+* Any attempts to read uninitialized parts of the new target image will redirect the read operation to the source image.
+* If data is written to an uninitialized part of the target image, Ceph automatically deep-copies the corresponding blocks from the source.
+
+**Syntax:**
+
+```
+rbd migration prepare SOURCE_POOL_NAME/SOURCE_IMAGE_NAME TARGET_POOL_NAME/TARGET_IMAGE_NAME
+```
+
+**Example:**
+
+```
+rbd migration prepare source_pool1/source_image1 target_pool1/target_image1
+```
+
+We can initiate the import-only live migration process by running
+the ``rbd migration prepare`` command with the ``--import-only`` and
+either ``--source-spec`` or ``--source-spec-path`` options, passing a JSON file
+hat describes how to access the source image data.
+
+1. Create a JSON file:
+
+```
+[ceph: root@rbd-client /]# cat testspec.json
+ {
+   "type": "raw",
+    "stream": {
+        "type": "s3",
+        "url": "https:hots_ip:80/testbucket1/image.raw",
+        "access_key": "Access key",
+        "secret_key": "Secret Key"}
+}
+```
+
+2. Prepare the import-only live migration process:
+
+**Syntax:**
+
+```
+rbd migration prepare --import-only --source-spec-path "JSON_FILE" TARGET_POOL_NAME/TARGET_IMAGE_NAME
+```
+
+**Example:**
+
+```
+[ceph: root@rbd-client /]# rbd migration prepare --import-only --source-spec-path "testspec.json" target_pool/traget_image
+```
+
+We can check the status of the migration with the ``rbd status`` command:
+
+```
+[ceph: root@rbd-client /]# rbd status target_pool/target_image
+Watchers: none
+Migration:
+source: {"stream":{"access_key":"RLJOCP6345BGB38YQXI5","secret_key":"oahWRB2ote2rnLy4dojYjDrsvaBADriDDgtSfk6o","type":"s3","url":"http://10.74.253.18:80/testbucket1/image.raw"},"type":"raw"}
+destination: targetpool1/sourceimage1 (b13865345e66)
+state: prepared
+```
+
+#### Step 2: Execute Migration
+
+Once the preparation is complete, Ceph starts deep copying all existing data
+from the source image to the target image.
+
+* The migration runs in the background, so applications can use the new target image immediately.
+* Any new write operations are stored only on the target image, ensuring a seamless transition.
+
+**Syntax:**
+
+```
+rbd migration execute TARGET_POOL_NAME/TARGET_IMAGE_NAME
+```
+
+**Example:**
+
+```
+rbd migration execute target_pool1/target_image1
+```
+
+### Step 3: Finalizing the Migration
+
+Once the data has been fully transferred, you need to commit or abort the migration.
```

Comment:
s/Once/After/

This is a personal preference of mine, but I find "Once" more ambiguous than "After" in contexts like this.